libraries/devsec_ssh.rb

```diff
@@ -0,0 +1,220 @@
+# encoding: utf-8
```

Comment:
Module/class name is `DevSec::SSH`, as there is no similar path in place, I called the file `devsec_ssh` to allow a similar matching between file name and class/module name